activerecord/test/cases/adapter_test.rb

```diff
@@ -690,6 +690,21 @@ def teardown
         end
       end
 
+      unless current_adapter?(:SQLite3Adapter)
```

Comment:
I couldn't find a way to get an id for a SQLite3 connection, so I'm opting to test with just MySql2 and PostregSQL.